packages/validator/src/services/indices.ts

```diff
@@ -76,6 +76,14 @@ export class IndicesService {
   private async pollValidatorIndicesInternal(pubkeysHex: PubkeyHex[]): Promise<ValidatorIndex[]> {
     const pubkeysHexToDiscover = pubkeysHex.filter((pubkey) => !this.pubkey2index.has(pubkey));
 
+    pubkeysHex.forEach((pubkey) => {
+      if (this.pubkey2index.has(pubkey)) {
+        this.logger.info("Validator status: ", {status: "active"});
```

Comment:
do we know that the status is `active` here? This is related to https://github.com/ChainSafe/lodestar/issues/5158, we currently do not keep track of the validator status as far as I am aware